_data/program_2021.yml

```diff
@@ -0,0 +1,48 @@
+- date: "2021-03-24"
+  dateReadable: "Day 1: Wednesday, 24<sup>th</sup> March 2021"
+  tracks:
+    - { title: "TrainTrack", color: "#90be4e" }
+    - { title: "HackTrack", color: "#03a9f4" }
+  timeSlots:
+    - {
+        startTime: "08:00",
+        endTime: "09:00",
+        sessionTitle: "coffee break",
+        speaker,
+        sessionType,
+        slides,
+      }
+    - {
+        startTime: "09:00",
+        endTime: "10:00",
+        sessionTitle: "Best session event",
+        speaker: "Marie Curie",
+        sessionType,
+        slides: "google.com",
```

Comment:
This currently opens at openmrbenelux.github.io/page-program/google.com. Is there a way to format as hyperlink so that this does not happen? Will probably important in the future (this may be a noob question but I don't know if this is just a simple matter of adding `https://` to the link or not)

Reply:
yeah I suspect that `slides` in the yml should just be a valid URL and we use that when creating the table.